packages/devtools_app/lib/src/framework/framework_core.dart

```diff
@@ -37,6 +37,8 @@ typedef ErrorReporter = void Function(String title, Object error);
 
 final _log = Logger('framework_core');
 
+EditorThemeManager? themeManager;
```

Comment:
make this a static variable on the FrameworkCore class instead of a top level variable in the library.